README.md

```diff
@@ -213,13 +213,8 @@ We use containers to minimize the installation requirements for development and
 
 We use Docker's Compose feature to spin up all application resources together. Please make sure you have Docker Compose installed on your system. You can find documentation on doing so here: [Docker Compose](https://docs.docker.com/compose/install)
 
-If the following commands return sensible results then Docker and Docker Compose are installed:
-
-```bash
-docker && docker compose || docker-compose
-```
 
-(above just runs docker and docker-compose and checks if both work)
+#### Option 1: Using Podman (Recommended)
 
 NOTE: As of 3/29/24 we recommend using `Podman` and `Podman Desktop` instead of `Docker` and `Docker Desktop`. It will be our default development environment going forward.
 There are many guides to transitioning from `Docker` (Desktop) to `Podman` (Desktop), but here's a rough outline of our "golden path."
```

Comment:
Move instructions on how to change from docker to podman under the initial podman section

Reply:
Lines 219-236 should be an additional section under the main instructions for using podman. Its a rare situation to move between the two